src/lib/registration/widgets/checkbox_item.rb

```diff
@@ -0,0 +1,191 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+
+module Registration
+  module Widgets
+    # A plain Ruby object in charge to build an item "checkbox" representation to be used by
+    # {MasterDetailSelector} widget in a RichText widget.
+    #
+    # FIXME: give support for disabled items; see #icon
```

Comment:
Question, does exist the possibility to have _disabled_ addons? If so, when? If not, I think we could go ahead by now and add support for this later. What do you think?

Reply:
Again, thanks.

Now I have a better picture of this situation. I was requesting to postpone it because I thought that was a use case not longer available. Indeed, I think supporting _disabled_ items it's a nice to have for the widget. Now I know that actually it's a must.

Reply:
Done, see https://github.com/yast/yast-registration/pull/483/commits/b8a381b127c9b54746995c63ca9e077e745304c6